custom_components/sscpoe/config_flow.py

```diff
@@ -145,19 +166,281 @@ def login():
+
+                     # Check web accessibility for all candidates
+                    if web_candidates:
+                         await self._check_web_accessibility(web_candidates)
+    
+                     # Add web devices to local_devices
+                    for candidate in web_candidates:
+                         if candidate.get("web_token"):
+                             ip = candidate["ip"]
+                             # Find existing device with this IP
+                             src_dev = next((d for d in self.local_devices if d["ip"] == ip and d["add_method"] == "old"), None)
+                             if src_dev:
+                                 web_device = self._normalize_device_data({
+                                     "add_method": "web",
+                                     "ip": ip,
+                                     "sn": src_dev["sn"],
+                                     "V": src_dev["V"],
+                                     "Active_state": "active",
+                                     "web_token": candidate["web_token"],
+                                     "user_password": SSCPOE_LOCAL_DEF_PASSWORD,
+                                 })
+                                 self.local_devices.append(web_device)
+
+                    return await self.async_step_local_select()
+
+            bind_default = bind_iface
+            ttl_default = ttl
+    
+        schema_dict: dict = {
+            vol.Required(CONF_BIND_INTERFACE, default=bind_default): vol.In(iface_choices),
+            vol.Required(CONF_MCAST_TTL, default=ttl_default): NumberSelector(
+                NumberSelectorConfig(
+                    min=1,
+                    max=255,
+                    mode=NumberSelectorMode.BOX  # This ensures display as a text input field
+                )
+            ),
+        }   
+   
+        return self.async_show_form(
+            step_id="local_multicast",
+            data_schema=vol.Schema(schema_dict),
+            errors=errors,
+        )    
+    
+    async def _check_web_accessibility(self, devices: list[dict]) -> None:
+        """Check HTTP management accessibility with default password for devices."""
+        
+        for device in devices:
+            ip = device.get("ip")
+            if not ip:
+                continue
+                
+            try:
+                # Use the same method as in async_step_local_web
+                err, token = await self.hass.async_add_executor_job(
+                    SSCPOE_web_login, ip, SSCPOE_LOCAL_DEF_PASSWORD
+                )
+                if not err:
+                    # Add token for further use
+                    device["web_token"] = token
+                else:
+                    LOGGER.debug("SSCPOE HTTP access check failed for %s: %s", ip, err)
+            except Exception as e:
+                LOGGER.debug("SSCPOE HTTP access check exception for %s: %s", ip, e)
+                
+        return None
```

Comment:
Move it as check_web_accessibility to protocol.py
Note it is bad idea to try login with a default password.
Use web request cmd=100 instead w/o a token.
```suggestion
    async def check_web_accessibility(self, devices: list[dict]) -> list[dict]:
        """Check HTTP management accessibility."""
        res = []
        for device in devices:
            ip = device.get("ip")
            if not ip:
                continue
            try:
                j, err = await self.hass.async_add_executor_job(
                    SSCPOE_web_request, ip, None, 100
                )
                if j is not None:
                    res.append(device)
                else:
                    LOGGER.debug(f"SSCPOE WEB access check failed for {ip}: {err}")
            except Exception as e:
                LOGGER.debug(f"SSCPOE WEB access check exception for {ip}: {str(e)}")
        return res
```